scripts/install_third_party_libs.py

```diff
@@ -22,17 +22,22 @@
 import shutil
 import subprocess
 import zipfile
+
+from core import feconf
 from scripts import install_python_dev_dependencies
+
 from typing import Final, List
 
-install_python_dev_dependencies.main(['--assert_compiled'])
+if not feconf.OPPIA_IS_DOCKERIZED:
+    install_python_dev_dependencies.main(['--assert_compiled'])
+
+    from . import install_third_party  # isort:skip  pylint: disable=wrong-import-position, wrong-import-order
+    from . import pre_commit_hook  # isort:skip  pylint: disable=wrong-import-position, wrong-import-order
+    from . import pre_push_hook  # isort:skip  pylint: disable=wrong-import-position, wrong-import-order
+    from . import setup  # isort:skip  pylint: disable=wrong-import-position, wrong-import-order
+    from . import setup_gae  # isort:skip  pylint: disable=wrong-import-position, wrong-import-order
```

Comment:
if you are talking about `assert_compiled` flag, then it is not required in docker, since it checks that any deps hash is not changed within the server env, but since we are moving to docker, and all the deps will be installed there only, so we don't require this script